src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1019,7 +1019,21 @@ int RPCCall::fromCommandLine (const std::vector<std::string>& vCmd)
         }
         else
         {
-            auto const setup = setup_ServerHandler(getConfig(), std::cerr);
+            ServerHandler::Setup setup;
+            try
+            {
+                std::stringstream ss;
+                setup = setup_ServerHandler(getConfig(), ss);
+            }
+            catch(...)
+            {
+                // eat this
```

Comment:
Shrug, just I've usually ultimately been bitten when I use a catch all.

I'm coming from a Python perspective though, where exceptions are more
pervasive.

I don't know what exceptions this throws.